xcube_geodb_openeo/server/app/flask.py

```diff
@@ -38,57 +38,61 @@
     )
 )
 
+well_known = flask.Blueprint('well_known', __name__)
 api = flask.Blueprint('api', __name__, url_prefix=API_URL_PREFIX)
 
 ctx = AppContext(app.logger)
 
 
+@well_known.route('/.well-known/openeo')
+def get_well_known():
+    return capabilities.get_well_known(ctx.config)
+
+
 @api.route('/')
 def get_info():
-    return {
-        'name': __name__,
-        'version': __version__
-    }
+    return capabilities.get_root(
+        ctx.config, ctx.for_request(f'{flask.request.root_url}'
+                                    f'{api.url_prefix}')
+    )
 
 
-@api.route('/catalog')
-def get_catalog_root():
-    return catalog.get_root(
-        ctx.for_request(flask.request.root_url)
-    )
+@api.route('/conformance')
+def get_conformance():
+    return capabilities.get_conformance()
 
 
-@api.route('/catalog/collections')
+@api.route('/collections')
 def get_catalog_collections():
-    return catalog.get_collections(
-        ctx.for_request(flask.request.root_url)
-    )
+    return catalog.get_collections(ctx.for_request(f'{flask.request.root_url}'
+                                                   f'{api.url_prefix}')
+                                   )
 
 
-@api.route('/catalog/collections/<string:collection_id>')
+@api.route('/collections/<string:collection_id>')
 def get_catalog_collection(collection_id: str):
-    return catalog.get_collection(
-        ctx.for_request(flask.request.root_url),
-        collection_id
-    )
+    return catalog.get_collection(ctx.for_request(f'{flask.request.root_url}'
+                                                  f'{api.url_prefix}'),
+                                  collection_id
+                                  )
 
 
-@api.route('/catalog/collections/<string:collection_id>/items')
+@api.route('/collections/<string:collection_id>/items')
 def get_catalog_collection_items(collection_id: str):
     return catalog.get_collection_items(
-        ctx.for_request(flask.request.root_url),
+        ctx.for_request(f'{flask.request.root_url}'
+                        f'{api.url_prefix}'),
         collection_id
     )
 
 
 @api.route('/catalog/collections/<string:collection_id>/'
            'items/<string:feature_id>')
 def get_catalog_collection_item(collection_id: str, feature_id: str):
-    return catalog.get_collection_item(
-        ctx.for_request(flask.request.root_url),
-        collection_id,
-        feature_id
-    )
+    return catalog.get_collection_item(ctx.for_request(flask.request.root_url),
+                                       collection_id,
+                                       feature_id
+                                       )
```

Comment:
Our coding convention is like so:

Either you use
```suggestion
    return catalog.get_collection_item(ctx.for_request(flask.request.root_url),
                                       collection_id,
                                       feature_id)
```

or

```python
    return catalog.get_collection_item(
        ctx.for_request(flask.request.root_url),
        collection_id,
        feature_id
    )
```